modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/form_builder.jspf

```diff
@@ -15,14 +15,22 @@
 --%>
 
 <%
-String defaultLanguageId = LocaleUtil.toLanguageId(LocaleUtil.getSiteDefault());
-
 Locale[] availableLocales = {LocaleUtil.getSiteDefault()};
+String defaultLanguageId = LocaleUtil.toLanguageId(LocaleUtil.getSiteDefault());
+boolean hasDDMDocumentLibraryDataType = false;
 
 if (Validator.isNotNull(script)) {
 	try {
 		DDMForm ddmForm = DDMUtil.getDDMForm(script);
 
+		for (DDMFormField ddmFormField : ddmForm.getDDMFormFields()) {
+			String dataType = ddmFormField.getType();
+
+			if (dataType.equals("ddm-documentlibrary")) {
+				hasDDMDocumentLibraryDataType = true;
+			}
```

Comment:
Hi @evibudai, could you please change to: 

```
if (StringUtil.equals(ddmFormField.getType(), "ddm-documentlibrary")) {
	hasDDMDocumentLibraryDataType = true;

	break;
}
```

Thanks!

Reply:
@carolmariaabb  I changed it :)